doc/getting_started/setup_fpga.md

```diff
@@ -427,25 +427,17 @@ If the firmware load fails, try pressing the "USR-RST" button before loading the
 
 ## Debugging with JTAG
 
-### Tap Straps
-
-After bootstrapping the firmware, the TAP straps may need to be set.
-As of this writing, the FPGA images are typically programmed to be in the RMA lifecycle state, and the TAP straps are sampled continuously in that state.
-To connect the JTAG chain to the CPU's TAP, adjust the strap values with opentitantool.
-Assuming opentitantool has been built and that the current directory is the root of the workspace, run these commands:
-
-```sh
-./bazel-bin/sw/host/opentitantool/opentitantool --interface cw340 \
-        --exec "gpio write TAP_STRAP0 false" \
-        --exec "gpio write TAP_STRAP1 true" \
-        no-op
-```
 
 ### CW340 Board
 
 The CW340 supports JTAG-based debugging with OpenOCD and GDB via the FTDI Chip embedded on the board or via HyperDebug (if you connected one to your board above).
 No external JTAG adapter is needed.
 
+**Tap Straps**
```

Comment:
I cannot build the docs at the moment so I do not know what this renders as, but I believe it to be bold. I think this should either be a h4 heading `####`, or just a separate paragraph following from the first.